kotlinx-coroutines-core/build.gradle

```diff
@@ -17,6 +20,14 @@ if (rootProject.ext.native_targets_enabled) {
 
 apply from: rootProject.file("gradle/compile-js-multiplatform.gradle")
 
+// TODO: Wasm target names have been changed since 1.9.20, remove this after migration to 1.9.20
```

Comment:
Starting from line 33, there is ASCII diagram with source-sets structure and their dependencies.

Please add WASM/WASI there

Reply:
You mean Wasm/WASI. Fixed